man/systemd.exec.xml

```diff
@@ -2375,6 +2375,39 @@ RestrictNamespaces=~cgroup net</programlisting>
         <xi:include href="version-info.xml" xpointer="v233"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DelegateNamespaces=</varname></term>
+
+        <listitem><para>Delegates ownership of the given namespaces to the user namespace of the processes of
+        this unit. For details about Linux namespaces, see <citerefentry
+        project='man-pages'><refentrytitle>namespaces</refentrytitle><manvolnum>7</manvolnum></citerefentry>.
+        Either takes a boolean argument, or a space-separated list of namespace type identifiers. If false
+        (the default), the unit's processes will not have ownership over any namespaces created during setup
+        of the unit's sandboxed environment. If true, ownership of all namespaces created during setup of the
```

Comment:
of all namespace types (except for user namespaces, where the concept doesn't apply)